js/CADWorker/workerGlobals.js

```diff
@@ -0,0 +1,20 @@
+import { initOpenCascade } from "opencascade.js";
+
+export const workerGlobals = {
```

Comment:
Loading everything onto one object was a quick way of getting things working. It can probably be clean up to something a bit more elegant.